salt/backups/templates/backup_mysql.sh

```diff
@@ -17,7 +17,8 @@ PASSPHRASE={{ settings.duplicity_passphrase }} /usr/bin/duplicity \
           --s3-use-server-side-encryption {{ backupdir }} \
           --archive-dir {{ cachedir }} \
           --full-if-older-than 1W \
-          --allow-source-mismatch --tempdir /backups \
-          s3+http://odl-operations-backups/{{ settings.get('directory', 'mysql') }}/
+          --allow-source-mismatch --tempdir /backups/tmp/ \
+          s3+http://odl-operations-backups/{{ settings.get('directory', 'mysql') }}/ \
+          {{ settings.database }}/
```

Comment:
This line still needs to be addressed... I don't think that it is needed at all since each database is dumped to its own file already. As it stands right now, the syntax is invalid.